backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/flightleg/BpkFlightLeg.kt

```diff
@@ -0,0 +1,138 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.invisibleToUser
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.style.BaselineShift
+import androidx.compose.ui.text.withStyle
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun BpkFlightLeg(
+  departureArrivalTime: String,
+  description: AnnotatedString,
+  stopsInfo: String,
+  duration: String,
+  modifier: Modifier = Modifier,
+  nextDayArrival: String? = null,
+  highlightStopsInfo: Boolean = false,
+  operatedBy: String? = null,
+  warning: String? = null,
+  contentDescription: String? = null,
+  carrierLogoContent: @Composable (BoxScope.() -> Unit)? = null,
+) {
+  Box(
+    modifier = modifier
+      .fillMaxWidth()
+      .semantics(mergeDescendants = true) {
+        if (contentDescription == null) {
+          // prevent screen readers from reading this allowing a broader
+          // content description to be set on the parent node
+          invisibleToUser()
+        } else {
+          this.contentDescription = contentDescription
+        }
+      },
+  ) {
+    Row(modifier = Modifier.fillMaxWidth()) {
+      carrierLogoContent?.let {
+        Box(
+          modifier = Modifier.padding(top = BpkSpacing.Sm),
+          content = it,
+        )
+        Spacer(modifier = Modifier.width(BpkSpacing.Base))
+      }
+      Column(
+        horizontalAlignment = Alignment.Start,
+      ) {
+        BpkText(
+          text = buildAnnotatedString {
+            append(departureArrivalTime)
+            nextDayArrival?.let {
+              withStyle(
+                SpanStyle(
+                  color = BpkTheme.colors.textPrimary,
+                  fontSize = BpkTheme.typography.label3.fontSize,
+                  baselineShift = BaselineShift.Superscript,
+                ),
+              ) {
+                append(it)
+              }
+            }
+          },
+          color = BpkTheme.colors.textPrimary,
+        )
+        BpkText(
+          text = description,
+          style = BpkTheme.typography.caption,
+          color = BpkTheme.colors.textSecondary,
+        )
+        operatedBy?.let {
+          BpkText(
+            text = it,
+            style = BpkTheme.typography.caption,
+            color = BpkTheme.colors.textSecondary,
+          )
+        }
+        warning?.let {
+          BpkText(
+            text = it,
+            style = BpkTheme.typography.caption,
```

Comment:
this should be `label3` (as bold)

Reply:
thanks! changed it to `label3`
where did you see it specifying as bold? Visually it does look bold but I couldn't find anywhere in figma specifying it to be bold. In fact, the `CSS` section of the page shows `font-style: normal;`.
For educational purposes, how are font weights specified in these designs in Figma?